### ISSUE TITLE
release task return no results

### ISSUE THREAD

**Issue by sas05** [andres-montanez/Magallanes#353]
Hi, I'm facing one problem

with this line of code 

```
            $command = 'chown -R ' . $userGroup . ' ' . $currentCopy
                . ' && '
                . 'chown ' . $userGroup . ' ' . $releasesDirectory;
            $result = $this->runCommandRemote($command);
            if (!$result) {
                return $result;
            }
```

$result is null and deployment failed. 

platform: OSX
destination platform: ubuntu

**Comment by dn5** [comment]
Could you post your workspace Mage logs?

**Comment by Toflar** [comment]
Hey @sas05, I think you run in the same escaping issue as I did. Too many characters are being escaped. Try changing the `runCommandRemote()` as I outlined in https://github.com/andres-montanez/Magallanes/issues/350#issuecomment-278303557.

**Comment by andres-montanez** [comment]
Duplicate of #350
fixed in release 3.0.1, already available!

Regards!!
